cmd/gitops/beta/run/cmd.go

```diff
@@ -830,12 +830,39 @@ func runCommandWithoutSession(cmd *cobra.Command, args []string) error {
 						log.Successf("Reconciliation is done.")
 					}
 
+					portForwards := map[rune]watch.PortForwardShortcut{}
+
+					if dashboardInstalled {
+						portForwards[watch.PortForwardShortcutRunes[len(portForwards)+1]] = watch.PortForwardShortcut{
```

Comment:
this has the potential to cause an "index out of range" error if the size of the `portForwards` slice is bigger than `watch.PortForwardShortcutRunes`. I think we should constrain the number of shortcuts available to the user to be no higher than the number of runes available in `watch.PortForwardShortcutRunes`.

This is kind of an edge case but it would backfire sonner or later I suppose.

Reply:
Added a guard against it for now.

We'll need to add getting and sorting map keys when building the port forward shortcut list in a followup PR later. So, we can add a better guard with letter shortcuts (in addition to numeric shortcuts) supported as well.